testpilot/users/views.py

```diff
@@ -38,5 +43,15 @@ def list(self, request):
         })
 
 
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def user_retire(request):
+    user = request.user
+    result = {"id": user.id, "username": user.username}
+    logout(request)
+    user.delete()
```

Comment:
Sorry, I'm a bit slow. What happens when we "delete" a user?
Will we still have all their feedback and survey results? Or do we need to keep the user and just anonymize their account info?